docs/src/administration/users.md

```diff
@@ -1,65 +1,53 @@
+* After you add or remove a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment. You can redeploy by using the CLI command platform redeploy or the Redeploy button in the management console.
 
-Selecting a user to edit that user's permissions or delete the user's access to the project entirely.
 
-To add a new user, click the `Add` button.
 
-You can either grant the `Project admin` role to the user, which will give them `Admin` access to every environment in the project, or grant specific permissions for each environment type.
+| Role                     | Description |
+|------------------------- |-------------|
+|Viewer                    | Viewers can view all environments of this type.|
+|Contributor               | This role includes the viewer permissions. Contributors can push code and branch all environments of this type. By default, everyone with access equal or greater than Contributor can access the environment through SSH. To customize who can use SSH, [set the access key](https://docs.platform.sh/configuration/app/access.html) in your platform.app.yaml |
+|Admin                     | This role includes viewer and contributor permissions. Administrators can change settings and execute actions on all environments of this type. By default, everyone with access equal or greater than Contributor can access the environment through SSH. To customize who can use SSH [set the access key](https://docs.platform.sh/configuration/app/access.html) in your platform.app.yaml|
 
-After inviting a new user, users without a Platform.sh account receive an invitation email asking to confirm their details and register an account.
+{{< note theme="warning" title="Warning" >}}
+Once you add or remove a user from a project or environment, you must trigger a redeploy to propagate SSH access changes to each environment. Redeploy using the CLI command `platform redeploy` or in the management console, click **Redeploy**.
+{{< /note >}}
 
-In order to push and pull code (or to SSH into one of the project's environments), the user needs to add an SSH key or use the Platform CLI.
+### Add users from the console
 
-If the user already has an account, they receive an email with a link to access the project.
+1. Go to your console and select the project where you want to change the user permissions.
+2. Under **Settings**, click **Access** on the left-hand side of the console.
+Click **+Add**.
+3. Add the user details and choose the permissions.
+4. Click **Save**.
 
-## Manage users permissions with the CLI
+### Manage users permissions with the CLI
 
-You can use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md). to fully manage your users and integrate with any automated system.
+You can use the [Platform.sh CLI (Command Line Interface)](https://docs.platform.sh/development/cli.html) to fully manage your users and integrate with any automated system.
```

Comment:
The way the link was there before was correct. This way hard codes it, which means it doesn't get checked for accuracy and also doesn't change based on the environment where it is deploy. I would revert this change.